cob_control_mode_adapter/src/cob_control_mode_adapter_node.cpp

```diff
@@ -69,26 +69,33 @@ class CobControlModeAdapter
             ROS_ERROR("Parameter %s not set, shutting down node...", param.c_str());
             nh_.shutdown();
         }
+        
+        std::string joint_trajectory_controller_name = nh_.param("joint_trajectory_controller_name", "joint_trajectory_controller");
```

Comment:
The quotes are intended!
I will add a proper cast, since the template argument `std::string` cannot be derived from the second argument. I always forget that this function cannot handle `const char *`.